### DISCUSSION TITLE
Useing lazygit as my merge and diff tool?

### DISCUSSION THREAD

**Discussion by poperigby** [jesseduffield/lazygit#1457] [Q&A]
Is there a way to have lazygit launch when I run `git mergetool` or `git difftool`? Just setting it with `git config merge.tool lazygit`, doesn't work.

**Comment by jesseduffield** [comment]
That is not currently supported. Feel free to raise an issue for this, if one does not already exist :)

**Reply by poperigby** [reply]
Will do!
